runtime/js/99_main.js

```diff
@@ -665,7 +665,10 @@ delete Object.prototype.__proto__;
       cpuCount,
     } = runtimeOptions;
 
-    location.setLocationHref(locationHref);
+    if (locationHref != null) {
+      location.setLocationHref(locationHref);
+    }
```

Comment:
Have you encountered a situation where `locationHref` was null? It's expected that this value is always set for workers.

Reply:
Not inside the deno project, but i'm using deno crates in a projet which provides a "Service Worker"-like environment like  deno deploy or cloudflare workers.